affiliated/affiliated_package_review_process.md

```diff
@@ -82,41 +82,53 @@ posted on the pull request and also in a reply to your email.
 
 ## Reviewing affiliated packages
 
-*These instructions are for coordinators and reviewers*
-
-One of the coordination committee members will be in charge of the review
-process (not necessarily the same person for each package). This coordination
-committee member (hereafter the *coordinator*) will find someone to carry out
-the main review of the package (hereafter the *reviewer*). In this sense the
-role  of the coordinator is similar to that of a journal editor.  However
-this is only partially true, because the coordinator (speaking for the
-coordination committee as a whole) may add their views to the review, unlike a
-journal editor who is often strictly impartial on the review itself.
-The pool of available reviewers will be anyone who has an official role on
-[the Astropy team](http://www.astropy.org/team.html).
-
-The coordinator sends out an email to possible reviewers one at a time to find
-someone who is willing to review the package, using the template at the bottom
-of this document. Let them know at this point that the coordinator will also be
-doing a review and synthesizing the results into a single review. Once a
-reviewer accepts, the coordinator sends them an email with detailed review
-instructions, using the template at the bottom of this document. The coordinator
-then leaves a message on the pull request saying:
+*These instructions are for editors and reviewers.*
+
+The review process will be overseen and coordinated by individuals that are in
+the *Affiliated package review editor* role. This role
+may be filled by one or more individuals. For the rest of this section, we
+use the singular term *editor*, although multiple people may share this role
+(and choose how to do this). The
+editor will start off by finding someone to carry out the main review of the
+package (hereafter the *reviewer*). In this sense the role of the editor is
+similar to that of a journal editor (and hence the role name). However this is
+only partially true, because here the editor may add their views to the review,
+unlike a journal editor who is often strictly impartial on the review itself.
+The pool of available reviewers will be anyone who has an official role on [the
+Astropy team](http://www.astropy.org/team.html) or anyone who has participated
+in the Astropy project enough to be familiar with the project guidelines and requirements.
+In some cases, it may be necessary to have multiple reviewers depending on the
+expertise required, and if scientific or algorithmic expertise is needed which
+is not available in the project, reviewers can be requested from outside the
+project at the editor's discretion.
+
+The editor sends out an email to possible reviewers to find someone who is
+willing to review the package, using the template at the bottom of this
+document. Emails should only be sent to the next potential reviewer(s) - not
+many possible reviewers at the same time. The editor should let the potential
```

Comment:
Something like this would be clearer for me but I don't know if that is too much. Also, you would need to break that long sentence up if you decide to adapt it here. Thanks!

```suggestion
many possible reviewers at the same time; For instance, if the review needs one reviewer, a request should first be sent to Reviewer A only and then to Reviewer B after an X amount of time has elapsed without any reply from Reviewer A (recommended X is 4 weeks and an email telling Reviewer A that their service is no longer needed should also be sent out too). The editor should let the potential
```

Reply:
Thanks! I think this is getting too specific though - I think the editors should decide on such policies internally. Also in the above scenario I think you'd want to sent reminders to reviewers etc and that really is too much detail here. My point here originally was more that the editor shouldn't just email all core maintainers asking for a volunteer, but contact people individually.